client/coral-framework/utils/user.js

```diff
@@ -35,6 +37,41 @@ export const isBanned = user => {
   return get(user, 'state.status.banned.status');
 };
 
+/**
+ * isUsernameRejected
+ * retrieves boolean based on the username status
+ */
+
+export const isUsernameRejected = user => {
+  return get(user, 'state.status.username.status') === 'REJECTED';
+};
+
+/**
+ * isUsernameChanged
+ * retrieves boolean based on the username status
+ */
+
+export const isUsernameChanged = user => {
+  return get(user, 'state.status.username.status') === 'CHANGED';
+};
+
+/**
+ * getActiveStatuses
+ * returns an array of active status(es)
+ * i.e if suspension is active, it returns suspension
+ */
+
+export const getActiveStatuses = user => {
```

Comment:
We need to be careful introducing terms into the codebase especially when adding to the framework. What does an `active status` mean? What qualifies to be an `active status`? Is `username unset` an active status? is  `username set` an active status?

Reply:
Or maybe there is a better term to describe your intention?